TFUtil.py

```diff
@@ -2125,7 +2131,9 @@ def is_axis_dynamic(self, axis):
     axis_wo_batch = self.get_batch_axis_excluding_batch(axis)
     if axis_wo_batch in self.size_placeholder:
       return True  # not quite the same as get_dynamic_axes
-    assert isinstance(self.batch_shape[axis], int)
+    assert isinstance(self.batch_shape[axis], int), (
+      "the requested axis has neither a size_placeholder entry nor a fixed size"
+    )
```

Comment:
The closing bracket should be on the same line as the text. (See all other code. Keep code consistent.)
Also, you should add `"%s: the requested ..." % self)` for a better debug message.